pkg/exchange/coinbase/exchange.go

```diff
@@ -451,3 +452,24 @@ func (e *Exchange) queryOrderTradesByPagination(ctx context.Context, q types.Ord
 	}
 	return cbTrades, nil
 }
+
+// Utils
+func (e *Exchange) CancelOrdersBySymbol(ctx context.Context, symbol string) ([]types.Order, error) {
+	cbOrders, err := e.queryOrdersByPagination(ctx, symbol, []string{})
```

Comment:
why query before canceling?

Reply:
Since the coinbase API will only return order ids that are canceled, I need to do the queries to retrieve price, quantity, ...etc.
However, I think you get a point.
I should delete the orders first then fetch the data I need afterward.
In such way, the result should be consistent.